### PR TITLE
Make a Markdown version of INSTALL

### PR DESCRIPTION
**Pull request by larsnaesbye**
First version. Probably needs some cleanup afterwards.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+
+As said prevously Celestia provides several user interfaces, accordingly with
+interfaces it's built with it has different executable files installed to
+${CMAKE_INSTALL_PREFIX}/bin (e.g. with default CMAKE_INSTALL_PREFIX on
+Unix-like systems they are installed into `/usr/local/bin`).
+
+Here's the table which provides executable file names accordingly to interface:
+
+ Interface | Executable name
+-  | :-
```
Comment (375gnu) [inline INSTALL.md]: either an error or non-standard md extension

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+| ENABLE_GTK           | bool | **OFF   | Build legacy GTK2 frontend
+| ENABLE_QT            | bool | ON      | Build Qt frontend
+| ENABLE_WIN           | bool | ***ON   | Build Windows native frontend
+| ENABLE_THEORA        | bool | **ON    | Support video capture to OGG Theora
+| ENABLE_TOOLS         | bool | OFF     | Build tools for Celestia data files
+| ENABLE_TTF           | bool | ****OFF | Build with FreeType support
+| NATIVE_OSX_APP       | bool | OFF     | Support native OSX data paths
+
+Notes:
+ \* /usr/local on Unix-like systems, c:\Program Files or c:\Program Files (x86)
```
Comment (375gnu) [inline INSTALL.md]: in this block <br /> should be added at the end of lines

File: INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+
+Build GLEW from source (Homebrew's precompiled GLEW has wrong linking path):
+
+  `brew install glew --build-from-source`
+
+Clone the source and go to the source directory.
+
+Configure and build:
+
+  `mkdir build
```
Comment (375gnu) [inline INSTALL.md]: \`\`\` should be used here

File: INSTALL.md [hunk 4]
```diff
@@ -0,0 +1,328 @@
...
+
+Install optional packages:
+
+  `pacman -S mingw-w64-x86_64-fmt mingw-w64-x86_64-eigen3 mingw-w64-x86_64-luajit`
+
+Clone the source and go to the source directory.
+
+Configure and build:
+
+  `mkdir build`
```
Comment (375gnu) [inline INSTALL.md]: \`\`\` should be used here

File: INSTALL.md [hunk 5]
```diff
@@ -0,0 +1,328 @@
...
+* Check https://celestia.space/download.html.
+* Check https://bintray.com/celestia if it contains packages for your system.
+
+Stable version installation on Windows and OSX:
+* Check https://celestia.space/download.html.
+
+Development snapshots installation on Unix-like systems:
+### On Debian 10 and derived systems:
+
+  `curl https://download.opensuse.org/repositories/home:/munix9:/unstable/Debian_10/Release.key`
```
Comment (375gnu) [inline INSTALL.md]: must be 
```
  curl https://download.opensuse.org/repositories/home:/munix9:/unstable/Debian_10/Release.key | sudo apt-key add -
  echo "deb https://download.opensuse.org/repositories/home:/munix9:/unstable/Debian_10/ ./" | sudo tee /etc/apt/sources.list.d/celestia-obs.list
  sudo apt update && sudo apt install celestia
```

File: INSTALL.md [hunk 6]
```diff
@@ -0,0 +1,328 @@
+# Basic installation instructions
+
+Stable version installation on Unix-like systems (e.g. GNU/Linux or *BSD):
+* Check your OS repository for already built packages.
+* Check https://celestia.space/download.html.
+* Check https://bintray.com/celestia if it contains packages for your system.
+
+Stable version installation on Windows and OSX:
```
Comment (eyvallah) [inline INSTALL.md]: OSX should be replaced with macOS

Comment (larsnaesbye) [reply INSTALL.md]: Sure, althought it isn't in the text-only version either.

File: INSTALL.md (hunk 1 above)
Comment (larsnaesbye) [reply INSTALL.md]: Non-standard extension. But GitHub supports tables like that.
Else we need another way to make tables. Markdown supports HTML tables, but that's usually not elegant IMHO.

Comment (larsnaesbye) [reply INSTALL.md]: No sorry, it is an error.

**Comment by eyvallah** [comment]
Looks good to me, but please squash your commits

**Comment by larsnaesbye** [comment]
Dammit. I picked wrong before the push. Sorry. Closing this and opening another instead :-/

**Pull request closed without merging**